includes/models/model.llms.order.php

```diff
@@ -806,6 +806,18 @@ public function get_next_payment_due_date( $format = 'Y-m-d H:i:s' ) {
 
 	}
 
+	/**
+	 * Retrieve the timestamp of the next scheduled event for a given action
+	 *
+	 * @since [version]
+	 *
+	 * @param string $action Action hook ID. Core actions are "llms_charge_recurring_payment" and "llms_access_plan_expiration".
+	 * @return int|false Returns the timestamp of the next action as an integer or `false` when no action exist.
```

Comment:
```suggestion
	 * @return int|bool The timestamp for the next occurrence of a pending scheduled action, `true` for an async or in-progress action or `false` if there is no matching action.
```
looks like `as_next_scheduled_action` can return `true` for an async or in-progress action

https://github.com/woocommerce/action-scheduler/blob/v3.1.5/functions.php#L150

Reply:
didn't realize that. Thanks